// Copyright 2025 Google LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

syntax = "proto3";

package google.confidential_match;

import "protos/core/key_value.proto";

option java_package = "com.google.cm.core";
option java_outer_classname = "DataValueProto";

// A single data record representing a matched user used for internal storage.
message DataValue {
  // Metadata key-value pairs for the data value.
  // This can include type, user IDs, or any arbitrary information.
  repeated KeyValue associated_data = 1;
}
